template/.devcontainer/Dockerfile

```diff
@@ -54,9 +54,6 @@ RUN if [ -n "${GITHUB_TOKEN}" ]; then export GITHUB_TOKEN=${GITHUB_TOKEN}; fi  \
     --log-level debug \
     --export-file /home/${CONTAINER_USER}/export-esp.sh
 
-# Set default toolchain
-RUN rustup default {{ toolchain }}
-
```

Comment:
Setting the default value for `rustup` appears to irrelevant as the value is overwritten by `rust-toolchain.toml`.

```
rustup show
...
esp (overridden by '/home/esp/project-name/rust-toolchain.toml')
...
```

Removing so there is less confusion for new developers.